### PR TITLE
[system-probe] Conntrack fixes for all namespaces

### PR DESCRIPTION
**Pull request by hmahmood**
### What does this PR do?

Fixes some bugs in the conntrack namespace dump code, as well as adding back the listen on all namespaces flag on the netlink socket. Also adds an integration test for this flag.

### CODE REVIEW

File: pkg/network/netlink/consumer.go [hunk 1]
```diff
@@ -362,6 +363,10 @@ func (c *Consumer) initNetlinkSocket(samplingRate float64) error {
 		log.Debugf("rcv buffer size for netlink socket is %d bytes", size)
 	}
 
+	if err := c.socket.SetSockoptInt(unix.SOL_NETLINK, unix.NETLINK_LISTEN_ALL_NSID, 1); err != nil {
```
Comment (p-lambert) [inline pkg/network/netlink/consumer.go:366]: Well, that explains it 😛 
Any thoughts about exposing this functionality behind a configuration flag?
The Conntrack code is arguably the main CPU hog on system-probe so I wonder if we this should be disabled by default.
I think most of of the Conntrack events we'll receive for different namespaces will be discarded as they won't contain translations (except for things like Istio)

**Review by p-lambert — COMMENTED** [review]
Side note: there shouldn't be a change on `pkg/ebpf/bytecode/tracer-ebpf.go`

**Comment by hmahmood** [comment]
> Side note: there shouldn't be a change on `pkg/ebpf/bytecode/tracer-ebpf.go`

I am not sure why there is a change. I am using the latest version of the VM, and updated all dependencies, etc., but I still get the changes.

File: pkg/network/netlink/consumer.go (hunk 1 above)
Comment (hmahmood) [reply pkg/network/netlink/consumer.go:366]: Yes, I have been thinking about doing that. Given that the istio use case is the only one at this point, I think a config flag is a good idea. Let me add that.

**Review by p-lambert — APPROVED** [review]
We talked IRL about opening a follow-up PR adding the configuration param

**Pull request merged**